src/nameserv/pmi/pmi_nameserv.c

```diff
@@ -109,162 +61,3 @@ int MPID_NS_Free(MPID_NS_Handle * handle_ptr)
     MPL_UNREFERENCED_ARG(handle_ptr);
     return 0;
 }
-
-#if 0
```

Comment:
just realized this removal got lumped into this commit. will split out once tests finish.